helm/helm.go

```diff
@@ -138,7 +138,7 @@ func templateChart(ctx *ankh.ExecutionContext, chart ankh.Chart, ankhFile ankh.A
 	// Load `values` from chart
 	_, valuesErr := os.Stat(files.AnkhValuesPath)
 	if valuesErr == nil {
-		if _, err := util.CreateReducedYAMLFile(files.AnkhValuesPath, currentContext.Environment); err != nil {
+		if _, err := util.CreateReducedYAMLFile(files.AnkhValuesPath, context.EnvironmentClass); err != nil {
 			return "", fmt.Errorf("unable to process ankh-values.yaml file for chart '%s': %v", chart.Name, err)
```

Comment:
Really glad we didn't name that file `ankh-environment-values.yaml` cause we don't need to rename it now! 😌